meta-ibm/meta-witherspoon/recipes-phosphor/fans/phosphor-fan/witherspoon/phosphor-fan-control@.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Phosphor Fan Control Daemon
+Requires=max31785-hwmon-helper@ahb-apb-bus\x401e78a000-i2c\x2dbus\x40100-max31785\x4052.service
```

Comment:
If you leave off the Requires line, then when phosphor-fan-control starts it won't always try to start the helper.

Reply:
It still will regardless since that helper service is a oneshot and goes dead once it completes, so its not available to know for this service to start after.